scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -1049,24 +1050,34 @@ private void jobsRecovered(Collection<InternalJob> jobs) {
     }
 
     void getProgressAndPingTaskNode(RunningTaskData taskData) {
-        if (!jobs.canPingTask(taskData) ||
+        if (!jobs.canPingTask(taskData) || taskData.isRestarting() ||
             taskData.getPingAttempts() > PASchedulerProperties.SCHEDULER_NODE_PING_ATTEMPTS.getValueAsInt()) {
             return;
         }
 
         InternalTask task = taskData.getTask();
         try {
             int progress = taskData.getLauncher().getProgress();//(2)
+            taskData.resetFirstTaskLauncherFailureTime();
             //get previous inside td
             if (progress != task.getProgress()) {
                 task.setProgress(progress);//(1)
                 //if progress != previously set progress (0 by default) -> update
                 listener.taskStateUpdated(taskData.getUser(),
-                                          new NotificationData<TaskInfo>(SchedulerEvent.TASK_PROGRESS,
-                                                                         new TaskInfoImpl((TaskInfoImpl) task.getTaskInfo())));
+                                          new NotificationData<>(SchedulerEvent.TASK_PROGRESS,
+                                                                 new TaskInfoImpl((TaskInfoImpl) task.getTaskInfo())));
             }
         } catch (Throwable t) {
-            tlogger.debug(task.getId(), "TaskLauncher is not accessible, checking if the node can be reached.", t);
+            if (taskData.getFirstTaskLauncherFailureTime() < 0) {
```

Comment:
maybe you could add method to RunningTaskData, like 'taskLauncherNeverFailed()
'